guardrails/validator_base.py

```diff
@@ -331,7 +331,11 @@ def validate_stream(
             validation_result.validated_chunk = chunk_to_validate
         if isinstance(validation_result, FailResult):
             if validation_result.error_spans is None:
-                validation_result.error_spans = [ErrorSpan(0, len(chunk_to_validate))]
+                validation_result.error_spans = [
```

Comment:
i have no idea how this was green and got merged because these args are def kwargs and not positional. confirmed w @nichwch this should be fine

Reply:
I saw this same issue this morning and didn't realize it was addressed in this PR.  Feel free to override during the merge if necessary.  I think the only difference is the `reason` property.

Reply:
no worries. glad its fixed. 👍🏼

Reply:
i think its a fallback anyways so almost any message is an acceptable message besides the hard error it was throwing before